pkg/discovery/service.go

```diff
@@ -56,6 +58,7 @@ func New(logger *zap.Logger, ol *overlay.Cache, kad *kademlia.Kademlia, config C
 
 		refreshLimit:       config.RefreshLimit,
 		refreshConcurrency: config.RefreshConcurrency,
+		antechamberLimit:   config.AntechamberLimit,
```

Comment:
why not have this as part of the request, like the original findNear limit?

Reply:
Not sure I understand what you mean

Reply:
It's part of the request here
https://github.com/storj/storj/pull/2626/files#diff-288a6a2c91f0ea201adc03ad9dbeb188R65

Reply:
Ok never mind